cmd/crictl/events.go

```diff
@@ -48,6 +48,16 @@ var eventsCommand = &cli.Command{
 			return cli.ShowSubcommandHelp(c)
 		}
 
+		switch format := c.String("output"); format {
+		case "json", "yaml":
+			if len(c.String("template")) > 0 {
+				return fmt.Errorf("template can't be used with %q format", format)
+			}
+		case "go-template":
```

Comment:
should we confirm that the `template` is defined?

Reply:
Yes, I believe that would make sense. Although I don't see that there is extensive validation in other commands. Would it be beneficial to address that once and for all?

Reply:
I'd suggest to concentrate on one command at a time. If you want to improve other commands - please do. 

For this command specifically I think it is important to fail on validation early as it will wait for events and only fail trying to format first of them. It will be quite annoying